code/modules/reagents/chemistry/reagents/alcohol.dm

```diff
@@ -1458,3 +1458,58 @@
 	drink_desc = "A sawed-off cola bottle filled with Fernet Cola. You can hear cuarteto music coming from the inside."
 	taste_description = "low class heaven"
 	remove_nutrition = 1
+
+/datum/reagent/consumable/ethanol/gimlet
+    name = "Gimlet"
+    id = "gimlet"
+    description = "A sharp cocktail dating back to the 19th century. Gin and lime, nothing else."
+    color = "#DEF8AB" // rgb (222, 248, 171)
+    alcohol_perc = 0.3
+    drink_icon = "gimlet"
+    drink_name = "Gimlet"
+    drink_desc = "There are debates on whether this drink should be half gin and half lime, or three parts gin and one part lime. All you know is, it's alcohol."
+    taste_description = "sharpness"
+
+/datum/reagent/consumable/ethanol/sidecar
+    name = "Sidecar"
+    id = "sidecar"
+    description = "A citrus cocktail of cognac, lemon and orange."
+    color = "#D7A61E" // rgb (215, 166, 30)
+    alcohol_perc = 0.4
+    drink_icon = "sidecar"
+    drink_name = "Sidecar"
+    drink_desc = "You can smell the citrus from here!"
+    taste_description = "smooth cognac and tart citrus"
+
+/datum/reagent/consumable/ethanol/whiskey_sour
+    name = "Whiskey Sour"
+    id = "whiskeysour"
+    description = "A tantalizing mixture of whiskey, sugar, lemon juice...and egg whites?"
```

Comment:
Also for this and the drink description for the 'Arnold Palmer', you probably want an extra space after the ellipses.
```suggestion
	description = "A tantalizing mixture of whiskey, sugar, lemon juice... and egg whites?"
```

Reply:
also git is angry now and they failed the PR checks, f u g